internal/codespaces/api/api.go

```diff
@@ -646,7 +646,7 @@ type startCreateRequest struct {
 	Machine            string `json:"machine"`
 	VSCSTarget         string `json:"vscs_target,omitempty"`
 	VSCSTargetURL      string `json:"vscs_target_url,omitempty"`
-	PermissionsOptOut  bool   `json:"devcontainer_permissions_opt_out"`
+	PermissionsOptOut  bool   `json:"multi_repo_permissions_opt_out"`
```

Comment:
Is this backwards compatible?

Reply:
Yes the API still accepts the old param.

Reply:
Nice! yeah we'll probably want to do that for another CLI release or two ❤️